concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Luangthongkum-2019-341-1	1	all, all gone, whole		
+Luangthongkum-2019-341-2	2	ant	587	ANT
+Luangthongkum-2019-341-3	3	arrive	1387	ARRIVE
+Luangthongkum-2019-341-4	4	arrow, dart	977	ARROW
+Luangthongkum-2019-341-5	5	ascend, go up, rise	705	GO UP (ASCEND)
```

Comment:
3207 GO UP OR RISE (ONESELF) would be a bit more general (because of the rise in there, which does not really fit the 705 concept definition relating to climbing/walking)

Reply:
3207 GO UP OR RISE (ONESELF) does not include the rising of the sun, though, which is likely meant here due to the opposite also being in the list.